app/ide-desktop/lib/dashboard/src/providers/LocalStorageProvider.tsx

```diff
@@ -27,7 +27,14 @@ export interface LocalStorageProviderProps extends Readonly<React.PropsWithChild
 /** A React Provider that lets components get the shortcut registry. */
 export default function LocalStorageProvider(props: LocalStorageProviderProps) {
   const { children } = props
-  const localStorage = React.useMemo(() => new LocalStorage(), [])
+  const [, rerender] = React.useState<object>({})
```

Comment:
please use the `useRefresh` hook (the purpose of `useState<object>({})` is not clear)